mypy/pyinfo.py

```diff
@@ -24,12 +24,16 @@ def getprefixes():
 
 def getsitepackages():
     # type: () -> List[str]
-    if hasattr(site, 'getusersitepackages') and hasattr(site, 'getsitepackages'):
-        user_dir = site.getusersitepackages()
-        return site.getsitepackages() + [user_dir]
+    res = []
+    if hasattr(site, 'getsitepackages'):
+        res.extend(site.getsitepackages())
+
+        if hasattr(site, 'getsitepackages') and site.ENABLE_USER_SITE:
```

Comment:
Shouldn't this be checking for `'getusersitepackages'`, not `'getsitepackages'`?

Reply:
Good catch! I opened https://github.com/python/mypy/pull/11468